.github/scripts/check_owners.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Usage: ./check_OWNERS.sh mydir/tasks/apply-mapping some/other/dir
```

Comment:
I would probably remove this. You already have usage further on and also, the first thing at the top here should be the description.